src/components/Highlighter/StringHighlighter/__tests__/StringHighlighter.spec.tsx

```diff
@@ -1,83 +1,100 @@
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
 import { StringHighlighter } from '../StringHighlighter';
 
 describe('StringHighlighter', () => {
+    let view;
     const string = 'We are looking for php, java and javascript developer';
     const searchTerms = [];
-    const ignoreDiacritics = true;
-    const ignoreCase = true;
-    // eslint-disable-next-line react/display-name, react/prop-types
     const highlightRenderer = ({ substring, ...props }) => <b {...props}>{substring}</b>;
-    // eslint-disable-next-line react/display-name, react/prop-types
-    const renderComponent = (props) => {
-        return mount(
-            <StringHighlighter
-                string={string}
-                searchTerms={searchTerms}
-                highlighterCoreOptions={(ignoreDiacritics, ignoreCase)}
-                highlightRenderer={highlightRenderer}
-                {...props}
-            />
-        );
+    const defaultProps = {
```

Comment:
Don't need to use const ignoreDiacritics = true; because it's true by default

Reply:
Yes, but it's not clear what we expect here. I think we should add 
 expect(screen.getByText(minDate.getFullYear())).toBeInTheDocument();
 expect(screen.getByText(maxDate.getFullYear())).toBeInTheDocument();